bigquery-storage/api/BigQueryStorage.Samples/Resources/customer_record.proto

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// The BigQuery Storage API expects protocol buffer data to be encoded in the
```

Comment:
Wow, really? Eek, that's pretty grim. We should check on that - proto3 now has optional fields, so it should be able to check that way.

Reply:
Thanks for pointing this out. I reused the same proto as used in Python sample. I checked with proto3 and it worked just fine. I have updated the proto.